Add a more complete explanation for global usage of the toast builder (#308)
melt-ui/melt-ui: src/docs/content/builders/toast.md

## Usage

> Unlike most builders, the toast is not component-based. Instead, it provides a global functionality
> that can be accessed from anywhere in your application. To accomplish this, it is recommended that
> you call the builder function `createToaster` inside the lib directory. This will make the function
> accessible to all components in your application.

Reviewer: ```suggestion
Unlike most builders, the toast is not component-based. Instead, it provides a global functionality
that can be accessed from anywhere in your application. To accomplish this, it is recommended that
you create a global component that is called on the root of your application.
```

> The first step is to create a `Toast` component that will be used to render toast notifications. We

Reviewer: I'd call it Toaster tbh

Author: Yes now that you say it, Toaster seems better!
